Doc/deprecations/pending-removal-in-3.16.rst

```diff
@@ -1,18 +1,42 @@
 Pending Removal in Python 3.16
 ------------------------------
 
+* :mod:`builtins`:
+
+  * Bitwise inversion on Boolean types, ``~True`` or ``~False``
```

Comment:
This is an ultra nit, but after skimming the docs, I don't know if Boolean needs to be capitalized. It doesn't look like we consistently do that elsewhere.

Reply:
Yeah, we're definitely not consistent. A rough search shows it's about 50-50:

```console
❯ rg -s "\bboolean\b" Doc | wc -l
      87

❯ rg -s "\bBoolean\b" Doc | wc -l
      78
```

We should pick something and put it in our [style guide](https://devguide.python.org/documentation/style-guide/). I opened https://github.com/python/docs-community/issues/129 about this, but we don't need to wait for that bikeshed before merging this :)

Reply:
OK, let's do lowercase: https://github.com/python/docs-community/issues/129#issuecomment-2340283261 

```suggestion
  * Bitwise inversion on boolean types, ``~True`` or ``~False``
```